questions.js

```diff
@@ -16,21 +31,52 @@ function sumLargest(numbers) {
 // Question 4
 function filterStrings(strings) {
   // Your code here
+  let filter = [];
+  for (let i = 0; i < strings.length; i++){
+    for (let j = i +1; j < strings.length; j++){
+      if (strings[j] > 3){
+        filter.push(strings[i]);
+      }
+    }
+  }
+  return filter;
 }
 
 // Question 5
 function filterEven(numbers) {
   // Your code here
+  let onlyEvens = [];
+  for (let i = 0; i < numbers.length; i++){
+    if (numbers[i] % 2 === 0){
+      onlyEvens.push(numbers[i]);
+    }
+  }
+  return onlyEvens;
 }
 
 // Question 6
 function pluck(objects, key) {
   // Your code here
+  let specificKey = [];
+  for (let i = 0; i < objects.length; i++){
+    specificKey.push(objects[i][key]);
+  }
+  return specificKey;
 }
 
 // Question 7
 function isPalindrome(str) {
```

Comment:
Logic is not correct for this

Reply:
for which question? Question 06 or Question 07??